src/types.deno.ts

```diff
@@ -114,11 +114,11 @@ export class InputFile {
                     "Reading files by path requires a Deno environment",
                 );
             }
-            const file = await Deno.open(data);
-            return iterateReader(file);
+            using file = await Deno.open(data);
+            return file.readable;
```

Comment:
I think there's no need to use the standard library; perhaps using ReadableStream would be better.